weather/README.md

```diff
@@ -0,0 +1,82 @@
+| temp_min           | Forecasted lowest temperature in degrees Celsius (°C) of the day  | 6                                                                  |
+| precip_prob_day    | Forecasted chance of precipitation before noon in percentage (%)  | 30                                                                 |
+| precip_proc_night  | Forecasted chance of precipitation after noon in percentage (%)   | 0                                                                  |
+| text_summary_day   | Detailed forecast summary before noon                             | A mix of sun and cloud with 30 percent chance of showers. High 16. |
+| text_summary_night | Detailed forecast summary after noon                              | Cloudy periods. Low 6.                                             |
+| date_pulled        | Day of when the data is pulled                                    | 05/03/2023                                                         |
+
+
+## Data Pipeline
+
+The data pipeline runs at 11:30 pm daily on airflow with the DAG `pull_weather`. There are four tasks: 1) `no_backfill` 2), `pull_prediction`, 3) `pull_historical_city`, and 4) `pull_historical_airport`. All tasks are run by `weather_bot`. 
+
+![image](https://user-images.githubusercontent.com/46324452/235770699-275ea663-5035-4799-984b-5eb0e09878b1.png)
+
+1) `no_backfill`
+
+Uses `LatestOnlyOperator` that disable downstream tasks for backfill. This is set as an upstream for `pull_prediction` as forecast data cannot be backfilled.
+
+2) `pull_prediction`
+
+Runs script `prediction_import.py` which uses package `env_canada` to pull City of Toronto's forecast data of the next 5 days and insert into `weather.prediction_daily`. 
+
+2) `pull_historical_city`
+
+Runs script `historical_scrape.py` which uses package [Beautiful Soup](https://www.crummy.com/software/BeautifulSoup/bs4/doc/) to parse the HTML content returned from [request](https://docs.python-requests.org/en/master/user/quickstart/#make-a-request) for City of toronto. The day before execution date's data will be pulled and inserted into `weather.historical_daily_city`.
+
+
+3) `pull_historical_airport`
+
+Runs script `historical_scrape.py` which uses package [Beautiful Soup](https://www.crummy.com/software/BeautifulSoup/bs4/doc/) to parse the HTML content returned from [request](https://docs.python-requests.org/en/master/user/quickstart/#make-a-request) for Toronto Pearson Airport. The day before execution date's data will be pulled and inserted into `weather.historical_daily_airport`.
+
+## Backfill Data
+
+As mentioned before, only historical data can be backfilled. Other than backfilling from airflow's cli, you can also use the script `backfill_historical.py` for when you backfill a lot of dates (since you will have to backfill day by day in airflow). Note: Make sure you have a `db.cfg` in your home folder, and you have `weather_admins` permissions.
```

Comment:
Cannot find the file 'backfill_historical.py' on this branch. Is this section out of date?

Reply:
sorry I forgot to push it :meow_dio: